package.json

```diff
@@ -32,6 +32,7 @@
     "d3plus-color": "^0.6.2",
     "d3plus-common": "^0.6.39",
     "d3plus-shape": "^0.14.11",
+    "d3plus-text": "^0.9.33",
```

Comment:
We are not using `d3plus-text` directly, so this should not be part of the dependencies. Delete your `node_modules` folder and `package-lock.json`, then run `npm i`. That should grab the latest d3plus-text as a dependency of d3plus-shape (without requiring the need for a `d3plus-shape` release)